lxd/identities.go

```diff
@@ -277,7 +277,9 @@ func getIdentities(d *Daemon, r *http.Request) response.Response {
 		return response.InternalError(fmt.Errorf("Failed to unescape path argument: %w", err))
 	}
 
-	if authenticationMethod != "" {
+	if authenticationMethod == "current" {
```

Comment:
Lets have a constant defined for this used in both call site and here so we can clearly see what this is used for.

Reply:
Oh I see its just used by the client package. OK don't worry then.